build/serve.js

```diff
@@ -116,6 +116,7 @@ gulp.task('spawn-backend', ['backend', 'kill-backend'], function() {
       [
         `--apiserver-host=http://${conf.backend.apiServerHost}`,
         `--port=${conf.backend.devServerPort}`,
+        `--heapster-host=http://${conf.backend.heapsterServerHost}`,
```

Comment:
1 solution
It is possible run heapster inside cluster using kubectl command. Here you can find yaml files for it:
https://github.com/kubernetes/heapster/tree/master/deploy/kube-config/standalone
We need gulp task to get kubectl and second to create heapster.

2 solution 
Add heapster to hyperkube image and publish it as new huperkube-heapster image.